addon/mixins/touch-action.js

```diff
@@ -1,16 +1,19 @@
 import { get, defineProperty, computed } from '@ember/object';
 import Mixin from '@ember/object/mixin';
 import { isHTMLSafe, htmlSafe } from '@ember/string';
+import config from 'ember-get-config';
+
+const defaults = config.EmberHammertime || {};
 
 const FocusableInputTypes = ['button', 'submit', 'text', 'file'];
 // Set this to `false` to not apply the styles automatically to elements with an `action`
-const TouchActionOnAction = true;
+const TouchActionOnAction = (typeof defaults.touchActionOnAction == 'undefined') ? true : defaults.touchActionOnAction;
```

Comment:
Don't use `typeof`, and use `===` e.g.:

`const TouchActionOnAction = (defaults.touchActionOnAction === undefined) ? true : defaults.touchActionOnAction;`